.github/workflows/regression-tests.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         os: [ubuntu-latest]
         compiler: [g++-10, g++-13]
         include:
+          - os: ubuntu-20.04
+            compiler: clang++-12
```

Comment:
clang++-12 is not installed by default on the `ubuntu-latest` image, but it is on `ubuntu-20.04`, so this is the easiest way to include it.